metrics/web/lib/common/presentation/metrics_theme/config/text_field_config.dart

```diff
@@ -3,21 +3,23 @@ import 'package:metrics/common/presentation/metrics_theme/config/color_config.da
 
 /// A class that holds the common text field styles for application themes.
 class TextFieldConfig {
-  static const TextStyle fieldTextStyle = TextStyle(
+  static const TextStyle lightFieldTextStyle = TextStyle(
```

Comment:
What about moving this style directly to the light theme? It seems that it's used there only in 1 place. 

Probably the same thing for darkFieldTextStyle?